internal/parser/scanner/rule_based_scanner_test.go

```diff
@@ -241,7 +284,7 @@ func _TestRuleBasedScannerWithRuleset(input string, ruleset ruleset.Ruleset, wan
 			assert.Equal(want[i].Col(), got[i].Col(), "Col doesn't match")
 			assert.Equal(want[i].Offset(), got[i].Offset(), "Offset doesn't match")
 			assert.Equal(want[i].Length(), got[i].Length(), "Length doesn't match")
-			assert.Equal(want[i].Type(), got[i].Type(), "Type doesn't match")
+			assert.Equal(want[i].Type(), got[i].Type(), "Type doesn't match, want %s, but got %s", want[i].Type().String(), got[i].Type().String())
```

Comment:
We don't need this explicity for the others, because only the type is displayed as
```
want 0x94
got 0x96
```
so I added the type names to the message. If there is no name for that type, it will always print `Type(123)`.

Reply:
Yes, this bothered me back then.Nice!